src/tests/ftest/launch.py

```diff
@@ -1014,6 +1014,22 @@ def run_tests(test_files, tag_filter, args):
                 avocado_logs_dir,
                 get_test_category(test_file["py"]))
 
+            valgrind_logs_dir = os.environ.get("DAOS_TEST_SHARED_DIR",
+                                               os.environ['HOME'])
+
+            print('DEBUG log:1017, valgrind_logs_dir=', valgrind_logs_dir)
+            print('DEBUG log:1016, avocado_logs_dir=', avocado_logs_dir)
+
+            # Archive remote valgrind log files
+            return_code |= archive_files(
+                "valgrind log files",
+                os.path.join(valgrind_logs_dir, "latest", "valgrind_logs"),
```

Comment:
This argument is the destination for the all of the remote files and should be a directory in the `avocado/test-results/<this _test>` directory to ensure it gets included in the Jenkins test results:
```suggestion
                os.path.join(avocado_logs_dir, "latest", "valgrind_logs"),
```